tensorflow/core/util/port.cc

```diff
@@ -64,13 +64,13 @@ bool GpuSupportsHalfMatMulAndConv() {
 
 // Returns whether oneDNN should be enabled or disabled by default.
 //   - Linux: Enabled by default for CPUs with neural network features.
-//   - Windows: Enabled by default.
+//   - Windows: Enabled by default on x86 CPUs
 inline bool DefaultOneDnnPolicy() {
 #if !defined(INTEL_MKL)
   return false;
 #elif defined(PLATFORM_GOOGLE)
   return true;
-#elif defined(PLATFORM_WINDOWS)
+#elif defined(PLATFORM_WINDOWS) && !defined(_M_ARM) && !defined(_M_ARM64)
```

Comment:
Would `PLATFORM_IS_X86` work for you?
```c++
#elif defined(PLATFORM_WINDOWS) && defined(PLATFORM_IS_X86)
```
Please `#include "tsl/platform/platform.h"` if necessary.

Reply:
Yes, PLATFORM_IS_X86 worked for us. I have committed the changes, please review if any further change is needed from our end. Thank you